appveyor.experimental.yml

```diff
@@ -48,7 +48,7 @@ install:
 - cmd: echo /logger:"%ProgramFiles%\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll">> Directory.Build.rsp
 - cmd: |-
     cd scripts
-    C:\\Python35\\python set_version_to.py -v %APPVEYOR_BUILD_VERSION% -t %APPVEYOR_BUILD_VERSION%
+    C:\\Python311\\python set_version_to.py -v %APPVEYOR_BUILD_VERSION% -t %APPVEYOR_BUILD_VERSION%
```

Comment:
This file is pretty outdated. It should be sync'ed with appveyor.yml. I committed a quick first draft: 0fa03a24bc8e6e69307c4314c9da017954601d43 in parallel.

Reply:
Great!